src/app/content/components/ContentExcerpt.tsx

```diff
@@ -16,16 +16,19 @@ import { getBookPageUrlAndParams } from '../utils/urlUtils';
 
 interface Props {
   content: string;
-  className: string;
+  className?: string;
   source: string | LinkedArchiveTreeSection;
+  forwardedRef?: React.Ref<HTMLElement>;
+  [key: string]: any;
```

Comment:
why this? if you're looking for things you can add to a div then extend the div component props